backend/djangoindia/api/views/event.py

```diff
@@ -13,6 +22,10 @@
 
 
 # Create your views here.
+<<<<<<< HEAD
+class EventAPIView(generics.GenericAPIView, ListModelMixin, CreateModelMixin, RetrieveModelMixin):
+    queryset = Event.objects.all()
+=======

```

Comment:
This happened because you haven't resolved merge conflicts correctly.

Reply:
Yeah !! I am solving it